modules/mcagar/src/main/ts/ephox/mcagar/api/pipeline/ApiChains.ts

```diff
@@ -57,13 +58,13 @@ const cSetSelection = <T extends Editor>(startPath: number[], soffset: number, f
 
 const cSetSetting = <T extends Editor>(key: string, value: any): Chain<T, T> => {
```

Comment:
Should we deprecate these and add new cSetOption and setOption to mcagar and over time switch to that?

Reply:
It's already listed in the `cleanup` settings Jira that's meant to be done after everything is converted. We probably should just mention to rename it there as I really didn't want to touch McAgar here but had to unfortunately... At the moment it's still very much geared more at 4.x/5.x than at 6.x.

Reply:
I did this anyways since it was quick...